rootfs/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM gcr.io/google_containers/ubuntu-slim:0.5
+FROM ubuntu
```

Comment:
I'd be happy to. It looks as though `:latest` is what's recommended [by the ubuntu maintainers](https://hub.docker.com/_/ubuntu/):

> The ubuntu:latest tag points to the "latest LTS", since that's the version recommended for general use.

But it looks like they're identical so yeah, we should be explicit and use 16.04. I thought the two were maintained separately.